src/runtime/constructorbinding.cs

```diff
@@ -122,7 +126,7 @@ public static IntPtr tp_repr(IntPtr ob)
                 return self.repr;
             }
             MethodBase[] methods = self.ctorBinder.GetMethods();
-            string name = self.type.FullName;
+            string name = self.type.Value.FullName;
```

Comment:
If it's invalid then probably print something like `(missing {self.type.Name})` -- makes it possible to debug.

Reply:
This one was missing it's check.